configurations/create_csap_pool.py

```diff
@@ -28,14 +29,204 @@
+                flow_result_configurations=[
+                    # FCDO-EiE (ran late 2020 to early 2021)
+                    FlowResultConfiguration("csap_s10_demog", "location", "location"),
+                    FlowResultConfiguration("csap_s10_demog", "gender", "gender"),
+                    FlowResultConfiguration("csap_s10_demog", "age", "age"),
+                    FlowResultConfiguration("csap_s10_demog", "recently_displaced", "recently_displaced"),
+                    FlowResultConfiguration("csap_s10_demog", "livelihood", "livelihood")
                 ]
             )
         )
     ],
-    archive_configuration=None
+    csv_sources=[
+        # TODO: Include the recovery csvs from EU-PCVE s01
+    ],
+    coda_sync=CodaConfiguration(
+        coda=CodaClientConfiguration(credentials_file_url="gs://avf-credentials/coda-production.json"),
+        sync_config=CodaSyncConfiguration(
+            project_users_file_url="gs://avf-project-datasets/2021/EU-PCVE/s02/coda_users.json",
+            dataset_configurations=[
+                CodaDatasetConfiguration(
+                    coda_dataset_id="CSAP_location",
+                    engagement_db_dataset="location",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/mogadishu_sub_district"),
+                                                auto_coder=somali.DemographicCleaner.clean_mogadishu_sub_district,
+                                                coda_code_schemes_count=1),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/somalia_district"),
+                                                auto_coder=lambda text:
+                                                somali.DemographicCleaner.clean_somalia_district(text)
+                                                if somali.DemographicCleaner.clean_mogadishu_sub_district == Codes.NOT_CODED
+                                                else Codes.NOT_CODED,
+                                                coda_code_schemes_count=1),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/somalia_region"),
+                                                auto_coder=None,
+                                                coda_code_schemes_count=1),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/somalia_state"),
+                                                auto_coder=None,
+                                                coda_code_schemes_count=1),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/somalia_zone"),
+                                                auto_coder=None,
+                                                coda_code_schemes_count=1)
+                    ],
+                    ws_code_string_value="location"
```

Comment:
match value?

Reply:
Thanks - string_value in this case, but I'll switch to match value as soon as we update the test pipeline